gr-utils/python/modtool/modtool_add.py

```diff
@@ -110,7 +112,13 @@ def setup(self, options, args):
         if not options.license_file:
             self._info['copyrightholder'] = options.copyright
             if self._info['copyrightholder'] is None:
-                self._info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+                user = getpass.getuser()
+                git_user = subprocess.check_output('git config --global user.name', shell=True).strip()
+                copyright_candidates = (user, git_user, 'GNU Radio')
```

Comment:
If this is the only place `git_user` is used, I'd recommend to fail gracefully and silently when the git command fails.